modules/dynamic-provisioning-annotations.adoc

```diff
@@ -28,7 +28,7 @@ metadata:
 
 This enables any persistent volume claim (PVC) that does not specify a
 specific storage class to automatically be provisioned through the
-default storage class.
+default storage class. However, your cluster can have more than one StorageClass, but only one of them can be the default StorageClass.
```

Comment:
s/StorageClass/storage class 

per guidance in https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references

Reply:
Fixed! Thanks.